soh/src/code/audio_data.c

```diff
@@ -544,10 +544,17 @@ u8 gDefaultShortNoteGateTimeTable[] = {
 };
 
 AdsrEnvelope gDefaultEnvelope[] = {
+#ifdef IS_BIGENDIAN
+    { 1, 32000 },
+    { 1000, 32000 },
+    { -1, 0 },
+    { 0, 0 },
```

Comment:
keeping this in hex might be more readable than in decimal, although as int16_t, maybe the little endian one is wrong. should this be uint16_t for the type in `AdsrEnvelope`?

Reply:
There original data was big endian and stored as signed integers. See https://github.com/HarbourMasters/Shipwright/blob/39cc86c2608e2f75ace33fc7f43bc4f2ad743f1a/soh/src/code/audio_data.c#L547-L556
For little endian the values were converted to hex and manually swapped. I would've just kept the original decimal representation for big endian, but I can also change both to hex.

Reply:
It'd make sense for them to be consistent. You could just wrap the LE data in decimal in endian conversion macro.